build/media_source/com_templates/vue/pagebuilder.js

```diff
@@ -0,0 +1,34 @@
+import Vue from 'vue';
+import VueJsModal from 'vue-js-modal';
+import Vuetify from 'vuetify';
+import App from './components/app.vue';
+import Translate from './plugins/translate';
+import editposition from './components/settings/editposition.vue';
+import editgrid from './components/settings/editgrid.vue';
+import addcolumn from './components/settings/addcolumn.vue';
+import editColumn from './components/settings/editcolumn.vue';
+import AddGridModal from './components/modals/modal-add-grid.vue';
+import AddElementModal from './components/modals/modal-add-element.vue';
+import GridElement from './components/grid-element.vue';
+
+// Add the plugins
+Vue.use(Translate);
+Vue.use(VueJsModal);
+Vue.use(Vuetify);
```

Comment:
If we are using Vuetify only for sidebar you can create own.

Reply:
Nitish did already created an alternative to vuetify.